tests/postgres_tests/test_array.py

```diff
@@ -466,8 +466,8 @@ def test_group_by_order_by_select_index(self):
                 ],
             )
         sql = ctx[0]["sql"]
-        self.assertIn("GROUP BY 2", sql)
-        self.assertIn("ORDER BY 2", sql)
+        self.assertIn("GROUP BY 1", sql)
+        self.assertIn("ORDER BY 1", sql)
```

Comment:
Expected as the usage of `annotate` after `values` now _append_ instead of _insert_ at the beginning.